schutzbot/run_image_tests.sh

```diff
@@ -36,8 +36,8 @@ run_test_case () {
     echo "🏃🏻 Running test: ${TEST_NAME}"
     test_divider
 
-    # Set up the testing command.
-    TEST_CMD="$TEST_RUNNER -test.v ${IMAGE_TEST_CASES_PATH}/${TEST_CASE_FILENAME}"
+    # Set up the testing command with Azure secrets in the environment
+    TEST_CMD="env $(cat $AZURE_CREDS | xargs) $TEST_RUNNER -test.v ${IMAGE_TEST_CASES_PATH}/${TEST_CASE_FILENAME}"
```

Comment:
Could you explain what's going on with `env $(cat $AZURE_CREDS | xargs)`? Could we pass `$AZURE_CREDS` as-is, and let the test runners parse it where necessary? Why is this different from the other creds?

Reply:
I force-pushed a new version containing a clean-up and an explanation. I think passing secrets using environmental variables is way to go, e.g. `awscli` uses the same concept.